tests/tools/DEBUG_TriMesh.cpp

```diff
@@ -174,4 +173,41 @@ auto DEBUG_TriMesh::get_tmoe(const Tuple& t, Accessor<long>& hash_accessor)
 {
     return TriMeshOperationExecutor(*this, t, hash_accessor);
 }
+
+std::vector<std::vector<long>> DEBUG_TriMesh::get_sorted_simplicial_complex(const std::vector<Simplex> &sc) const
```

Comment:
The name of this function is not very intuitive. Please, either provide a better name or add a comment describing what this function does.